.github/workflows/generate-desktop-targets.yml

```diff
@@ -14,7 +14,7 @@ on:
   workflow_dispatch:
 
 env:
-  FLEET_DESKTOP_VERSION: 0.0.2
+  FLEET_DESKTOP_VERSION: 0.0.3
```

Comment:
One more nit, if this is the stable release, should it be `1.0.0`?

Reply:
Deployed it as 0.0.3. If everything is looking good, we can change it to 1.0.0 and push next week.